quacc/recipes/qchem/core.py

```diff
@@ -0,0 +1,627 @@
+        Effectively defaults to the lowest spin state given the molecular structure and charge.
+    xc
+        Exchange-correlation functional.
+        Defaults to wB97M-V.
+    basis
+        Basis set.
+        Defaults to def2-TZVPD.
+    scf_algorithm
+        Algorithm used to converge the SCF.
+        Defaults to DIIS, but for particularly difficult cases, GDM should be employed instead.
+    pcm_dielectric
+        Dielectric constant of the optional polarizable continuum impicit solvation model.
+        Defaults no None, in which case PCM will not be employed.
+    smd_solvent
+        Solvent to use for SMD implicit solvation model. Examples include "water", "ethanol", "methanol",
+        and "acetonitrile". Refer to the Q-Chem manual for a complete list of solvents available.
+        Defulats to None, in which case SMD will not be employed.
+    swaps
+        Dictionary of custom kwargs for the calculator. Must be formatted consistently with Pymatgen's
+        QChemDictSet's overwrite_inputs. For example: {"rem": {"symmetry": "true"}}.
+
+    Returns
+    -------
+    dict
+        Dictionary of results from quacc.schemas.ase.summarize_run
+    """
+
+    if pcm_dielectric is not None and smd_solvent is not None:
+        raise ValueError("PCM and SMD cannot be employed simultaneously! Exiting...")
+
+    true_charge, true_spin = true_charge_and_spin(atoms, charge, mult)
+
+    input_atoms = deepcopy(atoms)
+
+    overwrite_inputs = {"rem": {"method": xc}}
+
+    swaps = swaps or {}
+    for key in swaps:
+        if key not in overwrite_inputs:
+            overwrite_inputs[key] = {}
+            for subkey in swaps[key]:
+                overwrite_inputs[key][subkey] = swaps[key][subkey]
+
+    qchem_input_params = {
+        "basis_set": basis,
+        "scf_algorithm": scf_algorithm,
+        "qchem_version": 6,
+        "pcm_dielectric": pcm_dielectric,
+        "smd_solvent": smd_solvent,
+        "overwrite_inputs": overwrite_inputs,
+    }
+
+    if scf_algorithm.lower() == "gdm":
+        qchem_input_params["max_scf_cycles"] = 200
+
+    calc = QChem(
+        input_atoms=atoms,
+        cores=cores,
+        charge=charge,
+        spin_multiplicity=mult,
+        qchem_input_params=qchem_input_params,
+    )
+    atoms.calc = calc
+    atoms = run_calc(atoms)
+    return summarize_run(
+        atoms,
+        input_atoms=input_atoms,
+        charge_and_multiplicity=(true_charge, true_spin),
+        additional_fields={"name": "Q-Chem Static"},
+    )
+
+
+@ct.electron
+@requires(
+    Sella,
+    "Sella must be installed. pip install sella",
+)
+def relax_job(
+    atoms: Atoms,
+    cores: int | None = None,
+    charge: int | None = None,
+    mult: int | None = None,
+    xc: str = "wb97mv",
+    basis: str = "def2-svpd",
+    scf_algorithm: str = "diis",
+    pcm_dielectric: str | None = None,
+    smd_solvent: str | None = None,
+    swaps: dict | None = None,
```

Comment:
FYI: I've  changed it to `calc_swaps` in all the calculators to distinguish from the newly introduced `opt_swaps` and such.